src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/ConsentOrderApprovedDocumentService.java

```diff
@@ -160,7 +162,7 @@ public void stampAndPopulateContestedConsentApprovedOrderCollection(Map<String,
         populateContestedConsentOrderCaseDetails(caseData, stampedAndAnnexedDoc, pensionDocs);
     }
 
-    public void generateAndPopulateConsentOrderLetter(CaseDetails caseDetails, String authToken) {
+    public CaseDetails generateAndPopulateConsentOrderLetter(CaseDetails caseDetails, String authToken) {
```

Comment:
Could use the opportunity to create the downstream methods to use FinremCaseDetails instead of mapping in and out.